frontend/src/components/SceneLayout/SceneLayout.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+import '../../App.css';
+
+interface NavLink {
+    path: string;
+    label: string;
+}
+
+interface SceneLayoutProps {
+    title: string;
+    navLinks: NavLink[];
+    children: ReactNode;
+    backgroundImage: string;
+}
```

Comment:
Wie vorhin erwähnt, ich würde bei type bleiben und deine ganzen types in deiner zentralen types.ts verwalten